common/metrics/defs.go

```diff
@@ -535,14 +535,15 @@ const (
 	PersistenceErrBusyCounter
 
 	NumCommonMetrics
+	HistoryFailures
```

Comment:
NumCommonMetrics need to be the last enum.  Please declare the new ones before this.

Reply:
Why?

Reply:
Nm, I see the subsequent constants.